### PR TITLE
Fix expired subscription sync products issue

### PR DESCRIPTION
**Pull request by nadjaheitmann**

### CODE REVIEW

File: app/lib/actions/scc_manager/subscribe_product.rb [hunk 1]
```diff
@@ -35,7 +35,6 @@ def finalize
         scc_product = SccProduct.find(input[:scc_product][:id])
         product = ::Katello::Product.find(input[:product_id])
         scc_product.update!(product: product)
-        ::Foreman::Logging.logger('foreman_scc_manager').warn input[:scc_product]
```
Comment (m-bucher) [inline app/lib/actions/scc_manager/subscribe_product.rb]: Can you remove this in the commit it was introduced with?

File: db/migrate/20210209080708_add_subscription_valid_to_scc_repositories.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddSubscriptionValidToSccRepositories < ActiveRecord::Migration[5.2]
+  def change
+    add_column :scc_repositories, :subscription_valid, :boolean, null: true
```
[the remark this answers is not in the document]
Comment (nadjaheitmann) [reply db/migrate/20210209080708_add_subscription_valid_to_scc_repositories.rb]: Fixed.

File: app/lib/actions/scc_manager/subscribe_product.rb (hunk 1 above)
Comment (nadjaheitmann) [reply app/lib/actions/scc_manager/subscribe_product.rb]: Fixed.

**Comment by nadjaheitmann** [comment]
image

**Comment by nadjaheitmann** [comment]
Closed in favor of #61 (contains the changes of this PR)

**Pull request closed without merging**